src/shared/test/java/com/couchbase/lite/internal/ExecutionServicesTest.kt

```diff
@@ -260,8 +260,8 @@ class ExecutionServicesTest {
         try { assertTrue(finishLatch.await(5, TimeUnit.SECONDS)) }
         catch (ignore: InterruptedException) { }
 
-        // within 10% is good enough
-        assertEquals((t - 777) / 10, 0L)
+        // within 10% is good enough for MacOS (30% for windows)
+        assertEquals((t - 777) / 30, 0L)
```

Comment:
@bmeike I am getting 20 milli seconds extra in my machine.

Reply:
I think that is probably evidence that we need a revised execution strategy for Windows.  Plus or minus 30% is not really acceptable...

Reply:
... and, fwiw, I don't know that that code has ever run on an OSX machine.  Those tests run on Android devices.  They pass even on emulators...

Reply:
updated the assertion to really check for 10%, before it was checking for 1%